frameworks/kafka/universe/config.json

```diff
@@ -263,6 +263,10 @@
           "description": "Disk type to be used for storing broker data. See documentation. [ROOT, MOUNT]",
           "default": "ROOT"
         },
+        "disk_profile": {
```

Comment:
Just for information, how does this setting interact with the disk type? Are there combinations that are not supported?

Reply:
yes, ROOT volume with profile is not supported.

How can I do some validation here?

Reply:
I think there are ways to do it in the config schema itself, but at the very least we could add a validator for the scheduler.